middleware/auth.js

```diff
@@ -1,32 +1,27 @@
-const multer = require('multer')
-
-/**
- * Middleware that returns a 403 error if not logged in. 
- */
-function isLoggedIn(req, res, next) {
-    if (req.isAuthenticated()) next()
-    else next({ status: 403, message: 'Unauthorized' })
-}
-
-
-/** 
- * Uploads an avatar file
- */
-const upload = multer({
-    limits: {
-        fileSize: 1000000
-    },
-    fileFilter(req, file, cb) {
-        if (!file.originalname.match(/\.(jpg|jpeg|png)$/)) {
-            return cb(new Error('Please upload an image'))
-        }
-
-        cb(undefined, true)
-    }
-})
-
-
-module.exports = {
-    isLoggedIn,
-    upload
+const multer = require('multer')
+
+/**
+ * Middleware that returns a 403 error if not logged in. 
+ */
```

Comment:
I don't know what you're doing to basically replace the contents of a file, but this kinda thing has gotta stooooop.  It totally messes up git merges/merge conflict issues.